site/content/docs/01-structure.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Sapper app structure
+---
+
+This section is a reference for the curious. We recommend you play around with the project template first, and come back here when you've got a feel for how things fit together.
+
+If you take a look inside the [sapper-template](https://github.com/sveltejs/sapper-template) repo, you'll see some files that Sapper expects to find:
+
+```bash
+├ package.json
+├ src
+│ ├ routes
+│ │ ├ # your routes here
+│ │ ├ _error.svelte
+│ │ └ index.svelte
+│ ├ client.js
+│ ├ server.js
+│ ├ service-worker.js
+│ └ template.html
+├ static
+│ ├ # your files here
+└ rollup.config.js / webpack.config.js
+```
+
+When you first run Sapper, it will create an additional `__sapper__` directory containing generated files.
+
+You'll notice a few extra files and a `cypress` directory which relates to [testing](docs#testing) — we don't need to worry about those right now.
+
+> You *can* create these files from scratch, but it's much better to use the template. See [getting started](docs#getting-started) for instructions on how to easily clone it
+
+
+### package.json
+
+Your package.json contains your app's dependencies and defines a number of scripts:
+
+* `npm run dev` — start the app in development mode, and watch source files for changes
+* `npm run build` — build the app in production mode
+* `npm run export` — bake out a static version, if applicable (see [exporting](docs#exporting))
+* `npm start` — start the app in production mode after you've built it
+* `npm test` — run the tests (see [testing](docs#testing))
+
+
+### src
+
+This contains the three *entry points* for your app — `src/client.js`, `src/server.js` and (optionally) `src/service-worker.js` — along with a `src/template.html` file.
+
+#### src/client.js
+
+This *must* import, and call, the `start` function from the generated `@sapper/app` module:
+
+```js
+import * as sapper from '@sapper/app';
+
+sapper.start({
+	target: document.querySelector('#sapper')
```

Comment:
Are you planning on getting #645 into 0.26.0?

Reply:
nah, i think it needs some testing first. can be done relatively unobtrusively later